testingfacade/Docs/QuestionSchema.json

```diff
@@ -0,0 +1,177 @@
+                        "required": [
+                            "answer_id",
+                            "label",
+                            "description",
+                            "id",
+                            "answer_str"
+                        ],
+                        "properties": {
+                            "answer_id": {
+                                "description": "String, unique to each answer within a question. This is the value to be returned for validation."
+                            },
+                            "label": {
+                                "description": "Resource label as might be displayed on the NCuT"
+                            },
+                            "description": {
+                                "description": "Resource description as might be displayed on the NCuT"
+                            },
+                            "id": {
+                                "description": "Resource ID as might be displayed on the NCuT"
+                            },
+                            "answer_str": {
+                                "description": "String combining all the resource details for display on the testing facade"
+                            }
+                        },
+                        "additionalProperties": true
+                    }
+                ]
+            }
+        },
+        "time_sent": {
+            "description": "Float, time the question was sent"
+        },
+        "timeout": {
+            "description": "Float, time in seconds the test suite will wait to receive an answer to a particular question",
+            "default": 600
+        },
+        "url_for_response": {
+            "description": "URL of the test suite API endpoint to send the POST request with answers"
+        },
+        "answer_response": {
+            "description": "Null when received. Should be filled with answer_ids according to question type when POSTed back to url_for_response",
+            "default": "",
+            "anyOf": [
+                {
+                    "type": "null",
+                    "description": "Null entry when sent from test suite"
+                },
+                {
+                    "description": "A single answer string, for 'radio' type questions",
+                    "examples": [
+                        "answer_1"
+                    ]
+                },
+                {
+                    "description": "Multiple answer strings, for 'checkbox' type questions",
+                    "examples": [
+                        ["answer_1", "answer_5"]
+                    ]
+                }
+            ]
+        },
+        "time_answered": {
+            "description": "Null when received. Should be filled with float string of time answered when returned",
+            "default": ""
+        },
+        "metadata": {
```

Comment:
I get the need for spec/test/question specific metadata.
Including the schema for the IS-05 metadata is good, and I don't have a pragmatic suggestion for how to do it for other specs until we get there, so I guess we just keep expanding this schema as needed?